grammar.js

```diff
@@ -188,6 +189,12 @@ module.exports = grammar({
       $._assignment
     ),
 
+    environment_variable_export: $ => seq(
+      'export',
+      $.simple_variable_name,
+      optional($._assignment)
+    ),
```

Comment:
It looks like this won't handle multiple variables being exported, which is legal. What do you think about something like this:

```js
export_command: $ => seq(
  'export',
  repeat(choice(
    $.simple_variable_name,
    $.environment_variable_assignment
  ))
),
```

Again, it's a bit more general than what is actually allowed (because `environment_variable_assignment` allows array subscript assignment), but I think it would be useful to have a visible node representing each assignment, rather than having the variable name and the assigned expression free-floating.

I don't have a strong opinion on the naming, but I thought `export_command` might be good since `export` is literally a built-in command similar to `alias` or `cd` (as opposed to a completely separate syntactic construct in bash).